accepted/2024/PMUI-Readme-rendering.md

```diff
@@ -22,15 +22,29 @@ The PM UI will be updated to have tabs for the Package Details and the the READM
 The README will be available in all the PM UI tabs.
 It will also be displayed for both the solution level and project level package managers.
 
+The selected tab will be maintained between sessions and when switching packages. 
+If the README tab is removed, then the "Package Details" tab will automatically selected.
```

Comment:
This isn't a change from what's already shipped, right?

Reply:
> This isn't a change from what's already shipped, right?

Correct, just describing what's already implemented. 

> What does remove mean in this case? How do you remove the tab?

I updated the language to say "available". The availability of the tab is described in the README Tab section.